realm/build.gradle

```diff
@@ -138,3 +138,84 @@ tasks.preBuild {
     dependsOn compileAnnotations
     dependsOn buildApt
 }
+
+def examples = [
```

Comment:
My evil master-plan is to completely automate the generation of the distribution folder.
But right now it's nice to have the flexibility to call the example folders in any way, without having to follow a convention.